npm/vue/package.json

```
@@ -74,4 +74,4 @@
   "ciJobs": [
     "npm-vue"
   ]
-}
+}
```

```suggestion
}

```

Super small change, just that we usually keep a newline at the end of files and my guess would be that it was removed here because of running the release script locally